docs/reference/modules/cluster/remote-clusters-security.asciidoc

```diff
@@ -0,0 +1,50 @@
+[[remote-clusters-security]]
+=== Configure remote clusters with security
+To use {ccr} or {ccs} safely with remote clusters, enable security on all 
+connected clusters and configure Transport Layer Security (TLS) on every node.
+Configuring TLS security at the transport level is minimally required for
+remote clusters. For additional security, configure TLS at the
+<<security-basic-setup-https,HTTP level>> as well.
+
+All connected clusters must trust one another and be mutually authenticated at
+the TLS level on at least the transport layer. This means that the local cluster
```

Comment:
```suggestion
with TLS on the transport interface. This means that the local cluster
```

We can drop "at least" since cross cluster does not use the HTTP level.